### PR TITLE
chore(security-groups): Rename Security Groups to Firewalls.

### PR DESCRIPTION
**Pull request by duftler**
Staging this PR as we work on replacing 'security group' with 'firewall'.
Screen-grabs will be added once ui changes are ready.

### CODE REVIEW

**Comment by lwander** [comment]
What work needs to be done in the release to enable this?

**Comment by duftler** [comment]
None.

**Comment by duftler** [comment]
Working on updating the various images now.

**Comment by duftler** [comment]
@nhayes @dorbin This PR is ready for review. The security groups -> firewall renaming is being released in 1.8. We should have this ready to go when that release is pushed. I believe there was also an internal codelab that you were updating @nhayes? We should try to have that one ready to go at the same time as well.

File: guides/tutorials/codelabs/hello-deployment/index.md [hunk 1]
```diff
@@ -276,22 +276,23 @@ our app.
...
 
-### Create security group
+### Create firewall
 
-Now we will create a security group to allow access to our application. Spinnaker only allows you to
- attach ingress sources based on another security group, so we first must create a base security 
- group via the aws console to allow traffic on port 8080.
+Now we will create a firewall to allow access to our application. Spinnaker only allows you to
+ attach ingress sources based on another firewall, so we first must create a base security
+ group via the aws console (AWS Security Groups map to Firewalls in Spinnaker) to allow traffic
```
Comment (dorbin) [inline guides/tutorials/codelabs/hello-deployment/index.md]: Can you move the parenthetical phrase to the end of the sentence?

**Review by dorbin — APPROVED** [review]
I only found one nit to pick. Otherwise, lgtm.
Thanks for doing this, Matt.

**Review by avh121 — APPROVED** [review]
LGTM, and the internal codelab update is ready to go whenever.

File: guides/tutorials/codelabs/hello-deployment/index.md (hunk 1 above)
Comment (duftler) [reply guides/tutorials/codelabs/hello-deployment/index.md]: Done.

**Review by dorbin — APPROVED** [review]

**Comment by duftler** [comment]
@nhayes The codelab CL can now be submitted as well.

**Pull request merged**